### PR TITLE
Unify code blocks.

### PR DESCRIPTION
**Pull request by cyhsutw**
Unify code blocks of documents.

Summary:
1. replace tab-based code blocks with backtick ones (fenced code blocks)
2. add syntax highlighting for Swift and Objective-C
3. unify whitespace and indentions in code blocks
4. remove trailing whitespace

### CODE REVIEW

File: docs/AccessingCarePlanData/AccessingCarePlanData-template.md [hunk 1]
```diff
@@ -216,33 +217,34 @@ This sample reads all the activities from the store, and then iterates over the
...
+			OCKCarePlanEventResult *result = [[OCKCarePlanEventResult alloc] initWithQuantitySample:sample quantityStringFormatter:nil unitStringKeys:@{[HKUnit degreeFahrenheitUnit]: @"\u00B0F", [HKUnit degreeCelsiusUnit]: @"\u00B0C",} userInfo:nil];
+
+			[_store updateEvent:assessmentEvent withResult:result state:OCKCarePlanEventStateCompleted completion:^(BOOL success, OCKCarePlanEvent * _Nonnull event, NSError * _Nonnull error) {
+				NSAssert(success, error.localizedDescription);
+				}];
+			}];                                        
+		}];
+	}
+}             
+```
```
Comment (umerkhan-apple) [inline docs/AccessingCarePlanData/AccessingCarePlanData-template.md]: We potentially want to convert this code snippet to Swift.
I would recommend including it in this PR, or at least tracking it under Issues.
`OCKSample` can serve as a good reference if you choose to do it in this PR.

**Comment by umerkhan-apple** [comment]
@cyhsutw Thanks for the PR! Minor feedback. Let me know what you think.

**Comment by cyhsutw** [comment]
@umerkhan-apple Thanks for taking time to review the changes! 

I've tried to convert the example code to the Swift version in the example project and tested on my devices.

Would you please take a few minutes to review it?

**Comment by umerkhan-apple** [comment]
@cyhsutw Looks great! r=m3! 👍

**Pull request merged**